SingularityUI/app/components/taskSearch/TaskSearch.jsx

```diff
@@ -141,4 +160,18 @@ function mapDispatchToProps(dispatch) {
   };
 }
 
-export default connect(mapStateToProps, mapDispatchToProps)(rootComponent(TaskSearch, 'Task Search'));
+let firstLoad = true;
+
+function refresh(props) {
+  if (!firstLoad) {
```

Comment:
If you only want refresh called when the page is first loaded and not again in an interval, just use the optional `refreshInterval` bool in `rootComponent` to false. `RequestDetail` does this https://github.com/HubSpot/Singularity/blob/decaf/SingularityUI/app/components/requestDetail/RequestDetailPage.jsx#L103